cli.js

```diff
@@ -27,6 +27,7 @@ const cli = meow([`
 })
 
 const authOptions = {
+  clientId: 'bfec45dffc45ea593ead',
```

Comment:
Should this be added?

Reply:
Well a client ID has to be added or else ghauth 5 won't be supported. I copied the clientId used in another package created by you since that made the most sense.